test/powershell/Modules/Microsoft.PowerShell.Management/Resolve-Path.Tests.ps1

```diff
@@ -48,4 +48,8 @@ Describe "Resolve-Path returns proper path" -Tag "CI" {
             Pop-Location
         }
     }
+    It 'Resolve-Path -Relative should support user specified base paths' {
+        $Expected = Join-Path -Path .\ -ChildPath fakeroot
+        Resolve-Path -Path $fakeRoot -Relative -RelativeBasePath $testRoot | Should -BeExactly $Expected
```

Comment:
Shouldn't using both `-Relative` and `-RelativeBasePath` now be invalid?

Reply:
Yes. I forgot to change the test.